Samples/outlook-set-signature/README.md

```diff
@@ -31,7 +31,9 @@ For documentation related to this sample, see [Configure your Outlook add-in for
 
 ## Applies to
 
-- Outlook on Windows, and on the web.
+- Outlook
+  - Windows
+  - web browser
 
 ## Prerequisites
```

Comment:
The npm prereq is not needed with GitHub hosting. If someone wants to run localhost the steps are later, and should be self-evident npm would be needed for that particular scenario. I would just list a Microsoft 365 account (developer account) as the prereq.